sprints/dashboard/utils.py

```diff
@@ -256,17 +261,35 @@ def get_issue_fields(conn: CustomJira, required_fields: Iterable[str]) -> Dict[s
     return {field: field_ids[field] for field in required_fields}
 
 
-def get_spillover_issues(conn: CustomJira, issue_fields: Dict[str, str]) -> List[Issue]:
+def get_spillover_issues(conn: CustomJira, issue_fields: Dict[str, str], project: str = '') -> List[Issue]:
     """Retrieves all stories and epics for the current dashboard."""
     return conn.search_issues(
         **prepare_jql_query_active_sprint_tickets(
             list(issue_fields.values()),
             settings.SPRINT_STATUS_SPILLOVER,
+            project=project,
         ),
         maxResults=0,
     )
 
 
+def create_next_sprint(conn: CustomJira, sprints: List[Sprint], cell_key: str, board_id: int) -> None:
+    """Creates next sprint for the desired cell."""
+    sprints = filter_sprints_by_cell(sprints, cell_key)
+    last_sprint = sprints[-1]
+    end_date = parse(last_sprint.endDate)
+
+    future_next_sprint_number = get_sprint_number(last_sprint) + 1
+    future_name_date = (end_date + timedelta(days=1)).strftime('%Y-%m-%d')
+    future_end_date = end_date + timedelta(days=settings.SPRINT_DURATION_DAYS)
+    conn.create_sprint(
+        name=f'{cell_key}.{future_next_sprint_number} ({future_name_date})',
+        board_id=board_id,
+        startDate=end_date.isoformat(),
+        endDate=future_end_date.isoformat(),
+    )
```

Comment:
There are two cases I can think of here:
- missing data in the request,
- Jira server unavailable.

In the first case retrying wouldn't help and in the second I would be worried about looping this for retrying multiple times (e.g. if two people ran into such issue). This could be very confusing and create many new sprints, if not handled properly.